src/poliastro/core/angles.py

```diff
@@ -311,7 +310,16 @@ def M_to_F(M, ecc):
     This uses a Newton iteration on the hyperbolic Kepler equation.
 
     """
-    F0 = np.arcsinh(M / ecc)
+    if ecc < 1.25:
+        if -np.pi < M < 0 or M > np.pi:
```

Comment:
My recommendation: "Always use less-than" https://thehardcorecoder.com/2014/03/05/always-use-less-than/ :)

Reply:
Thanks! That's interesting to know :)